letsencrypt-auto-source/pieces/bootstrappers/rpm_python3_legacy.sh

```diff
@@ -2,6 +2,17 @@
 # number must be increased.
 BOOTSTRAP_RPM_PYTHON3_LEGACY_VERSION=1
 
+EnablePython36SCL() {
+  set +e
+  # Do nothing if Python 3.6 is already available
+  ! "$EXISTS" python3.6 > /dev/null 2> /dev/null || return
+  # Do nothing if SCL Python 3.6 is not installed
+  scl --list 2>/dev/null | grep -q rh-python36 || return
+  # Install SCL rh-python36
```

Comment:
nit: I wouldn't say we're installing it. I'd personally just say "enable" or expand this comment a bit to say we're setting environment variables like `PATH` so Python 3.6 can be found and used by the rest of certbot-auto.

Reply:
Done.